components/style/properties.mako.rs

```diff
@@ -2798,6 +2777,44 @@ pub mod longhands {
             }
         }
 
+        impl ToCss for SpecifiedValue {
+            fn to_css<W>(&self, dest: &mut W) -> text_writer::Result where W: TextWriter {
+                let mut iter = self.0.iter();
+                if let Some(filter) = iter.next() {
+                    try!(filter.to_css(dest));
+                } else {
+                    try!(dest.write_str("none"));
+                    return Ok(())
+                }
+                for filter in iter {
+                    try!(dest.write_str(" "));
+                    try!(filter.to_css(dest));
+                }
+                Ok(())
+            }
+        }
+
+        impl ToCss for SpecifiedFilter {
+            fn to_css<W>(&self, dest: &mut W) -> text_writer::Result where W: TextWriter {
+                match *self {
+                    SpecifiedFilter::Blur(value) => try!(write!(dest, "blur({:?})", value)),
```

Comment:
`{:?}` is not necessarily doing the right formatting here. Use `value.to_css(dest)` instead, like `hue-rotate()`. I made a PR into your PR’s branch https://github.com/Adenilson/servo/pull/1. Merging it should automatically update this one.